app/views/spam2/_flags.html.erb

```diff
@@ -110,7 +110,7 @@ $(document).ready(function () {
              <td style="height:35px !important;">
                <a class="btn btn-xs border-curve font-weight-bold btn<% if flag.status != 1 %>-success<% else %>-secondary disabled<% end %> publish" data-remote="true" href="/moderate/publish/<%= flag.id %>" ><i class="fa fa-check-circle fa-white"></i> Publish post</a>
                <a class="btn btn-xs border-curve font-weight-bold btn<% if flag.status != 0 %>-danger<% else %>-secondary disabled<% end %> spam" data-remote="true" href="/moderate/spam/<%= flag.id %>"><i class="fa fa-ban fa-white"></i> Spam post</a> 
-               <a class="btn btn-xs border-curve font-weight-bold btn-warning unflag" data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
+               <a data-toggle="tooltip" data-placement="top" title="Remove all flags" class="btn btn-xs border-curve font-weight-bold btn-warning unflag" data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
```

Comment:
```suggestion
               <a id="unflag-node" data-toggle="tooltip" data-placement="top" title="Remove all flags" class="btn btn-xs border-curve font-weight-bold btn-warning unflag" data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
```

Reply:
And then on line 46, add the following code:
`$('#unflag-node').tooltip({ boundary: 'window' })`

Reply:
Let's hope that works!

Reply:
It worked🎉🎉🎉🎉

Reply:
This is fantastic, finally! Thank you, Tilda 😄

Reply:
I noticed that the tooltip kept flickering when I hovered over it so I searched for that and this was the SO answer that helped https://stackoverflow.com/a/56762953/11953728